client/kubeconfig.go

```diff
@@ -0,0 +1,105 @@
+package client
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fatih/color"
+)
+
+const DEFAULT_KUBECONFIG_PATH = ".kubeconfig"
+
+type KubeconfigPathOption string
+
+// WithSilent will stop the function from printing information to the console.
+const WithSilent KubeconfigPathOption = "silent"
+
+// LookupKubeconfigPath checks if a kubeconfig file exists, and returns its path.
+//
+// It first checks the "STACC_KUBECONFIG" environment variable, followed
+// by looking in the current directory.
+//
+// Use the WithSilent KubeconfigPathOption to stop this function from
+// logging helpful information directly to the console.
+func LookupKubeconfigPath(options ...KubeconfigPathOption) (string, error) {
+	var kubeconfigPath string
+
+	useSilent := false
+	for _, option := range options {
+		switch option {
+		case WithSilent:
+			useSilent = true
+		default:
+			return "", fmt.Errorf("unknown option %q", option)
+		}
+	}
+
+	var useEnvVar bool
+	staccKubeconfigEnv := os.Getenv("STACC_KUBECONFIG")
```

Comment:
nit: `os.LookupEnv(key string) (string, bool)` gives you `useEnvVar` for free.

Reply:
Very true, will fix